awx/api/views/__init__.py

```diff
@@ -2234,9 +2234,13 @@ class InventorySourceNotificationTemplatesAnyList(SubListCreateAttachDetachAPIVi
 
     def post(self, request, *args, **kwargs):
         parent = self.get_parent_object()
-        if parent.source not in models.CLOUD_INVENTORY_SOURCES:
+        if parent.source not in models.compute_cloud_inventory_sources():
```

Comment:
This needs to be imported from `aws.main.utils.plugins` now:
```suggestion
        if parent.source not in compute_cloud_inventory_sources():
```